lldb/source/Target/Thread.cpp

```diff
@@ -814,6 +809,17 @@ bool Thread::ShouldStop(Event *event_ptr) {
     return false;
   }
 
+  // Call this after ShouldStopSynchronous.
+  ThreadPlan *current_plan;
+  if (auto plan = GetProcess()->FindDetachedPlanExplainingStop(event_ptr)) {
+    current_plan = plan.get();
+  } else {
+    current_plan = GetCurrentPlan();
+  }
+
+  // The top most plan always gets to do the trace log...
```

Comment:
Super-nit: While we are modifying this, we could replace the ellipsis with a `.`.